app/models/project/Page.scala

```diff
@@ -141,10 +189,18 @@ object Page {
         TypographicExtension.create()
       ))
 
-    (Parser.builder(options).build(), HtmlRenderer.builder(options).build())
+
+    (Parser.builder(options).build(), HtmlRenderer.builder(options)
+      .linkResolverFactory(linkResolver.get)
+      .build())
   }
 
-  def Render(markdown: String): Html = Html(htmlRenderer.render(markdownParser.parse(markdown)))
+  def Render(markdown: String)(implicit config: OreConfig): Html = {
+    // htmlRenderer is lazy loaded so linkResolver will exist upon loading
+    if (linkResolver.isEmpty)
+      linkResolver = Some(new ExternalLinkResolver.Factory(config))
```

Comment:
I've removed the use of getting the OreConfig out of the global state. Still don't like this solution but it works